### ISSUE TITLE
Reset and Simplify the Revision History

### ISSUE THREAD

**Issue by sthagen** [oasis-tcs/sarif-spec#637]
The revision history in the specification has become an elaborated document in itself.

Also, because of the long sequence of minimal changes, the value for the reader seams to not match the volume of the content.

In addition, we changed the format of the specification source to be easily comparable across commits / revisions for everyone (as compared to the large word file we used before).

Last, but not least, we should cut the revision history to start with the changes following the v2.1.0 Errata 01 status.

## Proposal

Simplify the revision history and remove all entries that point to v2.1.0 Errata 01 or earlier revisions.

**Comment by sthagen** [comment]
Rewrite the revision history appendix as:

```
<!--
---
toc:
  auto: false
  label: (Informative) Revision History
  enumerate: Appendix L.
---
-->
# (Informative) Revision History

| Revision                  | Date       | Editors                                       | Description |
|:--------------------------|:-----------|:-----------------------------|:---------------------------------------------------------------------------------------------------|
| sarif-v2.2-wd20240502-dev | 2024-05-02 | Craig Schlaman, Stacey Wray, and Stefan Hagen | Initial version 2.2.0, transcribed from v2.1.0 Errata 01 with minor corrections. |
| sarif-v2.2-wd20240605-dev | 2024-06-05 | Stacey Wray and Stefan Hagen.                 | Allowed embedded links in multiformatMessage.text (issue 471).                   |
```

Rendered similar to:

# (Informative) Revision History

| Revision                  | Date       | Editors                                       | Description |
|:--------------------------|:-----------|:-----------------------------|:---------------------------------------------------------------------------------------------------|
| sarif-v2.2-wd20240502-dev | 2024-05-02 | Craig Schlaman, Stacey Wray, and Stefan Hagen | Initial version 2.2.0, transcribed from v2.1.0 Errata 01 with minor corrections. |
| sarif-v2.2-wd20240605-dev | 2024-06-05 | Stacey Wray and Stefan Hagen.                 | Allowed embedded links in multiformatMessage.text (issue 471).                   |

**Comment by stacywray** [comment]
@sthagen Hello! My name is spelled Stacy (no e). Thanks!

**Comment by sthagen** [comment]
@stacywray sure, sorry for that.

**Comment by sthagen** [comment]
Implemented per #638.
